src/content/docs/browser/new-relic-browser/configuration/proxy-agent-requests.mdx

```diff
@@ -0,0 +1,82 @@
+---
+
+From [version 1.240.0](/docs/release-notes/new-relic-browser-release-notes/browser-agent-release-notes) and higher, the browser agent officially allows a proxy url to be set when determining either or both of:
+
+1. Where to fetch or lazy-load agent code.
+2. Where to send analytics.
+
+Today, many ad-blockers and security VPNs block client requests to third-party domains including New Relic's. As the number of internet users opting into the use of ad-blockers increase, customers will continue to lose observability on those devices. One solution would be to proxy those requests though an unblocked middleman. For example, `yoursite.com` can send its visitor analytics through its `nrbeacon.yoursite.com` proxy which would not be blocked by anti-adware being that it's considered a first-party (sub)domain.
+
+<Callout variant="important">
+  This feature is currently available for those using the copy-and-paste _or_ NPM installation methods. There is currently no UI or NerdGraph configuration options available. We are continuing to work on improving access to these and other configuration options.
+</Callout>
+
+## How to use a proxy [#use]
+
+The agent utilizes two configuration properties when determining where to fetch chunks or send data.
+
+For example, using copy and paste:
+
+```js
+window.NREUM.init = {
+    ...<other init properties>,
+    proxy: {
+        assets: <URL to fetch agent chunks>,
+        beacon: <URL to send analytics>
+    }
+}
+```
+
+For NPM, similarly just pass the `init` object with a proxy as part of any other configurations:
+
+```js
+new BrowserAgent({ init: { proxy: <object similar to above> } })
+```
+
+`init.proxy.assets` is used verbatim in the copy-and-paste installation to fetch agent chunks.
+
+* For example, the v1.240.0 Pro+SPA loader will require `nr-spa.___-1.240.min.js` file.
+* If this field is set to be `nrassets.abc.com` reverse proxy, provide this value as the exact URL, including the scheme and ending slash: `'https://nrassets.abc.com/'`.
+
+This is not relevant to NPM installed agents, as the main host project's bundler should take care of resources. `init.proxy.beacon` is used by both installation to determine the host to which data such as page view timings are sent. 
+
+* For example, if those payloads are routed through a `nrbeacon.abc.com` proxy, then that is the exact string value to use. 
+<Callout variant="important">
+  * Note that the scheme (e.g, `'https://'`) and ending slash **should be omitted** for this property. The port, as part of the host, is accepted.
+</Callout>
+
+
+Paths can also be used:
+```js
+NREUM.init = {
+    proxy: {
+        assets: 'https://exampleproxy.com/chunks/',
+        beacon: 'exampleproxy.com/ingest'
+    }
+}
+```
+<Callout variant="tip">
+  The browser agent team recommends using different endpoints for your proxies.
+</Callout>
```

Comment:
```suggestion
<Callout variant="tip">
The browser agent team recommends using different values for each proxy configuration. This may mean each configuration is a different subdomain or possibly each contains a different path. This will make configuring the proxy server a lot easier.
</Callout>
```